Dockerfile

```diff
@@ -34,7 +34,7 @@ ENTRYPOINT ["/sbin/tini", "--"]
 HEALTHCHECK --interval=1s --timeout=30s --retries=30 \
     CMD  wget --no-verbose --tries=5 --spider http://localhost:9090/health || exit 1
 
-CMD [ "npm", "start" ]
+CMD [ "npm", "run", "start:inspect" ]
```

Comment:
is this PR created for testing purpose ?

Reply:
Yes